src/index.html

```diff
@@ -14,9 +16,118 @@
             <li>Races</li>
         </ul>
     </nav>
+    <section class="main">
+
+        <div class="card">
+            <div class="card-img"><img class="pokeimg" id="pokeimg" alt="pokeimage"></img></div>
+            <div class="card-info">
+
+                <div class="cardtitleparent">
+                    <h2 class="cardtitle">Bulbasaur</h2>
+                    <p class="likebutton"></p>
+
+                </div>
+                <p class="likebuttoncounter">(Like Counter)</p>
+                <button class="commentbutton btn">Comment</button>
+                <button class="reservationbutton btn">Reservation</button>
+            </div>
+        </div>
+
+        <div class="card">
+            <div class="card-img"><img class="pokeimg" id="pokeimg" alt="pokeimage"></img></div>
+            <div class="card-info">
+
+                <div class="cardtitleparent">
+                    <h2 class="cardtitle">Bulbasaur</h2>
+                    <p class="likebutton"></p>
+
+                </div>
```

Comment:
- Please make this part dynamic.

Reply:
Thank for bringing that issue up, the instructions are not clear about that feature, but I implemented it.